randovania/games/dread/json_data/Ferenia.txt

```diff
@@ -1250,6 +1255,10 @@ Extra - asset_id: collision_camera_018
       Any of the following:
           Gravity Suit
           Heat/Cold Runs (Intermediate) and Cold Damage ≥ 100
+  > Event - Tunnel to Cold Room Grapple Block
+      Grapple Beam and Gravity Suit and Speed Booster and Speed Booster Upgrade ≥ 3
+  > Tunnel to Cold Room (Storm Missile Gate)
+      Gravity Suit and Speed Booster and Speed Booster Upgrade ≥ 3 and After Ferenia - Tunnel to Cold Room Grapple Block
```

Comment:
These are actually redundant. The connection to them from the Charge door would also handle being able to do it from above. Not only cause you just can't ever come from the charge door first anyway, but because no matter which door you do it from you will have enough room, and you must use one of the doors to even get to this area.

Reply:
If the Charge door gets randomized to something you can't open via Door Lock Rando, you won't have the space down below. I think my thought process for the logic was that you can technically ADC into that room without using ANY of the doors (old Plasma Skip UR route).

There's no logic for the ADC right now, but adding that in would allow a progression item to be placed down there that would then allow you to open one of the doors and get back out. It might be very rare for it to be used, but given that MOST of the logic is already there, it can't hurt.

Reply:
I thought the ADC only skipped the plasma door. is that not the case? Or do you mean doing an ADC from where that bladetrap is in the room prior?

Reply:
Ok, on second thought, the ADC connection is going to take way more work because of the need for speed upgrades just to charge it up, and the fact that there isn't a great node to use (I think).